18F/identity-idp
# Upgrade premailer-rails for Propshaft compatibility

## 🛠 Summary of changes

Upgrades `premailer-rails` gem for Propshaft compatibility, to resolve errors which can happen when running some specs in local development.

See changelog: https://github.com/fphilipe/premailer-rails/blob/main/CHANGELOG.md#v1120

Related Slack discussion: https://gsa-tts.slack.com/archives/C0NGESUN5/p1684257860591849

The reason that this wasn't caught previously is that CI runs `rake assets:precompile`, which triggers Propshaft to resolve assets statically rather than dynamically. Running `rake assets:precompile` would work locally, but this shouldn't be a requirement.

Follows #8387

## 📜 Testing Plan

- Remove any compiled assets `rm -rf public/assets public/packs app/assets/builds/*.css`
- Run a spec which involves a mailer `rspec spec/lib/tasks/review_profile_spec.rb:30`
- Observe that it passes
